kde-settings/kde-settings.spec

```diff
@@ -1,9 +1,9 @@
 # THIS SPECFILE IS FOR F18 ONLY!
-
 %if 0%{?qubes_builder}
 %define _sourcedir %(pwd)/kde-settings
-%define epoch %(cat epoch)
 %endif
+%{!?version: %define version %(cat version)}
```

Comment:
Hmm, where does that value come from? I see:

```
%package minimal
Summary: Minimal configuration files for KDE
Requires: %{name} = %{epoch}:%{version}-%{release}
```

In the file. Is it set elsewhere?